### PR TITLE
Update pt-PT

### PR DESCRIPTION
**Pull request by esibruti**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Closes #issue...
- Related #issue...

**Details of Changes**
Add details of changes here.
- Added new translations

### CODE REVIEW

File: Files/MultilingualResources/Files.pt-PT.xlf [hunk 1]
```diff
@@ -6,7 +6,7 @@
     </header>
     <body>
       <group id="FILES/STRINGS/EN-US/RESOURCES.RESW" datatype="resx">
-        <trans-unit id="SideBarOpenInNewTab.Text" translate="yes" xml:space="preserve">
+            <trans-unit id="SideBarOpenInNewTab.Text" translate="yes" xml:space="preserve">
```
Comment (yaichenbaum) [inline Files/MultilingualResources/Files.pt-PT.xlf:9]: Can you fix the indentation?

**Review by yaichenbaum — CHANGES_REQUESTED** [review]

File: Files/MultilingualResources/Files.pt-PT.xlf (hunk 1 above)
Comment (esibruti) [reply Files/MultilingualResources/Files.pt-PT.xlf]: You want this way?
https://github.com/files-community/Files/pull/7161/commits/db857b2889f0a1b974125f4dcbebd22e69cfff35

Comment (esibruti) [reply Files/MultilingualResources/Files.pt-PT.xlf]: Or is it with this modification?
https://github.com/files-community/Files/pull/7161/commits/ea814f427e2b428f04bf336f21a6df13d280056c

**Review by yaichenbaum — APPROVED** [review]
LGTM

**Pull request merged**